components/atomic/Button/IconButton/index.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+import { ComponentProps, forwardRef, MouseEventHandler } from "react";
+import clsx from "clsx";
+import IconComposer from "@rubin-epo/epo-react-lib/IconComposer";
+import { ScreenreaderText } from "@rubin-epo/epo-react-lib/styles";
+import styles from "./styles.module.css";
+
+interface IconButtonProps {
+  onClickCallback?: MouseEventHandler;
+  accessibleText?: string;
+  visibleText?: string;
+  isBlock?: boolean;
+  icon: string;
+  size?: number;
+  buttonProps?: ComponentProps<"button">;
+  svgProps?: ComponentProps<"svg">;
+  iconProps?: ComponentProps<"svg">;
+  className?: string;
+}
+
+const IconButton = forwardRef<HTMLButtonElement, IconButtonProps>(
+  (
+    {
+      accessibleText,
+      visibleText,
+      isBlock = false,
+      icon,
+      size,
+      onClickCallback,
+      buttonProps,
+      iconProps,
+      className,
+    },
+    ref
```

Comment:
the only change here, apart from moving into TypeScript, is adding a `ref` to the button